### PR TITLE
Add ef migrations

### PR DESCRIPTION
**Pull request by jvandaal**

### CODE REVIEW

File: src/ParcelRegistry.Projections.Extract/Migrations/20240517142535_AddCountOnParcelLink.cs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+namespace ParcelRegistry.Projections.Extract.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddCountOnParcelLink : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropPrimaryKey(
```
Comment (jvandaal) [inline src/ParcelRegistry.Projections.Extract/Migrations/20240517142535_AddCountOnParcelLink.cs:14]: @ArneD I don't know why this change is added to the migration? More like these below.

File: src/ParcelRegistry.Projections.Legacy/Migrations/20240517142953_AddCountOnParcelDetailAddress.cs [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+namespace ParcelRegistry.Projections.Legacy.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddCountOnParcelDetailAddress : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropPrimaryKey(
```
Comment (jvandaal) [inline src/ParcelRegistry.Projections.Legacy/Migrations/20240517142953_AddCountOnParcelDetailAddress.cs:14]: @ArneD same comment as before.

**Review by ArneD — APPROVED** [review]
